src/util/history.js

```diff
@@ -0,0 +1,37 @@
+import { hashHistory } from 'react-router'
+
+export default hashHistory
```

Comment:
It isn't used in the functions below, but it does provide a single place for all other files to import the same history singleton object. `react-router` provides both a `hashHistory` and a `browserHistory` object, though you need to use the same one throughout the application to make everything work. Rather than having to maintain the context concerning history we import I figured we could just always import from our history utility file and it would be the same.